corehq/apps/reports/util.py

```diff
@@ -491,7 +509,7 @@ def update_tableau_user(domain, username, role=None, groups=[]):
     ).get(username=username)
     if role:
         user.role = role
-    new_id = session.update_user(user.tableau_user_id, role=user.role, username=username)
+    new_id = session.update_user(user.tableau_user_id, role=user.role, username=(tableau_username(username)))
```

Comment:
nit: redundant parentheses 
```suggestion
    new_id = session.update_user(user.tableau_user_id, role=user.role, username=tableau_username(username))
```